source/agora/node/BlockStorage.d

```diff
@@ -568,7 +574,7 @@ public class BlockStorage : IBlockStorage
 
     private size_t readSizeT (size_t pos) @trusted
     {
-        ubyte[] data = this.read(pos, pos+size_t.sizeof);
+        ubyte[] data = this.read(pos, size_t.sizeof);
```

Comment:
This seems to be the bug, no ?

Reply:
I don't think it is "the" bug we are looking for.